src/testServer/routing.js

```diff
@@ -1,58 +1,60 @@
 'use strict';
 
-const { URL } = require('url');
 const databaseInterface = require('./dbInterface.js');
 
+const makeResponse = (code, data) => (
+  { code, data }
+);
+
 const authentificate = (login, password) => {
   if (!login || !password) return {
-    err: '<h1>&#127819Login and password should be specified</h1>',
+    err: '&#127819Login and password should be specified',
+    code: 401,
     account: null
   };
   const account = databaseInterface.getAccount(login);
-  if (!account) return {
-    err: '<h1>&#127819Incorrect login entered :(</h1>',
-    account: null
-  };
-  if (password !== account.password) return {
-    err: '<h1>&#127819Incorrect password entered :(</h1>',
+  if (!account || password !== account.password) return {
+    err: '&#127819Incorrect login or password :(',
+    code: 403,
     account: null
   };
-  return { err: null, account };
+  return { err: null, code: 200, account };
 };
 
-const profileHandler = (credentials, res) => {
+const profileHandler = credentials => {
   const login = credentials[0];
   const password = credentials[1];
-  const { err, account } = authentificate(login, password);
-  if (err) {
-    res.writeHead(403);
-    res.end(err);
-    return null;
-  }
-  return account;
+  const { err, code, account } = authentificate(login, password);
+  if (err) return { code, data: err };
+  return makeResponse(code, account);
 };
 
-const cardsHandler = (credentials, res) => {
-  const account = profileHandler(credentials, res);
-  if (!account) return null;
+const cardsHandler = credentials => {
+  const result = profileHandler(credentials);
+  const { code } = result;
+  if (code !== 200) return result;
+  const account = result.data;
   const cards = databaseInterface.getCards(account.id);
-  return cards;
+  return makeResponse(code, cards);
 };
 
-const transactionsHandler = (credentials, res) => {
-  const cards = cardsHandler(credentials, res);
-  if (!cards) return null;
+const transactionsHandler = credentials => {
+  const result = cardsHandler(credentials);
+  const { code } = result;
+  if (code !== 200) return result;
```

Comment:
It may be that code will be 200 but there will be no account (maybe some error).